epoxy-paging3/src/main/java/com/airbnb/epoxy/paging3/PagedDataModelCache.kt

```diff
@@ -122,6 +123,8 @@ internal class PagedDataModelCache<T : Any>(
         workerDispatcher = dispatcher
     )
 
+    val loadStateFlow: Flow<CombinedLoadStates> = asyncDiffer.loadStateFlow
```

Comment:
I think it's better to not have a backing field
```suggestion
    val loadStateFlow: Flow<CombinedLoadStates> get() = asyncDiffer.loadStateFlow
```